src/helper/remote_config/protocol/tuf/serializer.cpp

```diff
@@ -8,20 +8,37 @@
 #include <rapidjson/prettywriter.h>
 
 #include "../../../json_helper.hpp"
+#include "../../../utils.hpp"
 #include "../cached_target_files.hpp"
 #include "base64.h"
 #include "exception.hpp"
 #include "serializer.hpp"
 
 namespace dds::remote_config::protocol {
 
+// NOLINTNEXTLINE(fuchsia-statically-constructed-objects)
+static std::string runtime_id_generated;
```

Comment:
Statics are dangerous and here we don't need it. Each remote configuration client is unique to a service so you can make the fake `runtime_id` a member of the client

Reply:
Done